### PR TITLE
Remove Unused Settings - Part 2

### PR DESCRIPTION
**Pull request by stevenjcumming**
## Summary

This is part 2 of 3-ish

- Remove settings from `config/settings.yml` and `config/settings/test.yml`
- Checked using:
   - Dot notation
   - Hash access (`[]`, `.dig`, `fetch`)
   - Checked parent key

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/95565

## Testing done

- [ ] n/a

## Acceptance criteria

- [x]  Unused settings are removed from `config/settings` files

### CODE REVIEW

**Comment by stevenjcumming** [comment]
@randomsync @stephenBarrs @kanchanasuriya could either of you take a look at the these removed setting to see if any are being used by your team?

File: config/settings.yml [hunk 1]
```diff
@@ -947,9 +944,7 @@ vetext:
 hqva_mobile:
   url: "https://veteran.apps.va.gov"
   mock: false
-  key_path: /app/secret/health-quest.key
```
Comment (rmtolmach) [inline config/settings.yml:950]: You modified this 6 months ago [here](https://github.com/department-of-veterans-affairs/vets-api/pull/16561/files#diff-e769bbb8c1ba3711c5403b424ed9c218ffafba7f1890ee394717196f28ff4540). But yeah, I don't see it used anywhere.

File: config/settings.yml [hunk 2]
```diff
@@ -1571,33 +1560,17 @@ check_in:
   authentication:
     max_auth_retry_limit: 3
     retry_attempt_expiry: 604800 # 7 Days
-  chip_api:
```
Comment (rmtolmach) [inline config/settings.yml:1574]: 👍 looks like `chip_api_v2` is what's used now

**Review by rmtolmach — APPROVED** [review]
I also didn't see any of these used.

File: config/settings.yml (hunk 2 above)
Comment (randomsync) [reply config/settings.yml:1574]: yes, that's correct. the changes LGTM for check_in settings.

**Review by randomsync — APPROVED** [review]

**Comment by stephenBarrs** [comment]
The `hqva_mobile` settings are related to an app that was never deployed. The [code still exists ](https://github.com/department-of-veterans-affairs/vets-api/tree/master/modules/health_quest) so the config should remain until it is all removed at the same time.  Changes look good to me!

**Review by stephenBarrs — APPROVED** [review]

**Pull request merged**